tests/Auth0Test.php

```diff
@@ -334,6 +343,10 @@ public function testThatRenewTokensSucceeds()
         $_SESSION['auth0__state'] = '__test_state__';
 
         $this->assertTrue( $auth0->exchange() );
+
+        $this->assertArrayNotHasKey('auth0__nonce', $_SESSION);
+        $this->assertArrayNotHasKey('auth0__state', $_SESSION);
+
         $auth0->renewTokens(['scope' => 'openid']);
```

Comment:
This failed in https://github.com/auth0/auth0-PHP/pull/434/commits/647aaed2baaf5274ccc0ad2bece434d8d6685f48 as expected but now passes with the changes in this PR.